web/app/view/State.js

```diff
@@ -63,9 +63,15 @@ Ext.define('Traccar.view.State', {
             renderer: function (value, metaData, record) {
                 if (record.get('attribute') === 'alarm') {
                     metaData.tdCls = 'view-color-red';
+                } else if (record.get('name') === Strings.positionAddress && !value) {
+                    return '<a href="#">' + Strings.stateShowAddress + '</a>';
```

Comment:
It is really hard to add `onclick` handler here. (or my knowledge is not enough)
But found on sencha forums tip to handle cell click instead of `a` element.